server.py

```diff
@@ -43,31 +55,32 @@
         statement = fl.read()
     questions[i] = Question(output=output, statement=statement)
 
+
 @app.route("/")
 def changePath():
     return redirect("/dashboard")
 
+
 @app.get("/dashboard")
 def dashboard():
     return template("dashboard.html", contests=contests)
 
+
 @app.get("/contest/<code>/<number>")
-def contest(code,number):
+def contest(code, number):
```

Comment:
we should not be able to see contest if it is not open

Reply:
```
if contests[code].start_time > datetime.datetime.now():
        return "The contest had not started yet."
```
These lines in the code already do that.

Reply:
I have not added a handler in for the questions pattern yet though.

Reply:
umm. that's in another api no?

Reply:
oh, sorry didnt check the position, will do it right away. This is in the questions handler

Reply:
Done.